vere/http.c

```diff
@@ -2451,16 +2477,15 @@ _proxy_ward_resolve(u3_warc* cli_u)
   hin_u.ai_socktype = SOCK_STREAM;
   hin_u.ai_protocol = IPPROTO_TCP;
 
+  //  XX why the conditional?
```

Comment:
Looks like it can be created without setting a host?

Reply:
It's always created without setting a host. I think this may have written with an eye to retrying failed resolutions, but I don't remember.